src/components/Dialog/TextDialog/QuestionDialog.vue

```diff
@@ -0,0 +1,101 @@
+<!--
+  ブラウザ版の質問ダイアログ。
```

Comment:
ここは名実ともに普通の質問ダイアログになりましたね！！！素晴らしい！！！
```suggestion
  質問ダイアログ。
```